config/defaults.env

```diff
@@ -32,3 +32,7 @@ OTEL_EXPORTER_OTLP_ENDPOINT=https://collector.opensafely.org
 
 # WARNING: care needs to be taken if changing this. See playbook.md
 DATABASE_ACCESS_NETWORK=jobrunner-db
+
+# RAP Controller
+DJANGO_CONTROLLER_ALLOWED_HOSTS='*'
```

Comment:
Doesn't this want to be more restricted for prod? Will it be called at localhost while the django app is inside the backend?

Reply:
No, at `controller:8000`, right? Can/should we set allowed hosts to just `controller`

Reply:
I'm not sure it really buys us anything here and it just introduces coupling between the Docker Compose config and the rest of the system. When we move the Controller out we could consider setting it. Although even then we've tended to use the nginx config to enforce the hostname.